cypress/integration/wards/bookingAVisit.js

```diff
@@ -0,0 +1,97 @@
+describe("As a ward staff, I want to schedule a virtual visit so that patients can speak with their loved ones.", () => {
+  before(() => {
+    // reset and seed the database
+    cy.exec(
+      "npm run dbmigratetest reset && npm run dbmigratetest up && npm run db:seed"
+    );
+  });
+
+  it("allows a ward staff to book a virtual visit", () => {
```

Comment:
What do we think about splitting this test up into the form validations and the happy path (booking a visit without issues)? That way we can see if there are issues with just the form validation or vice versa without the other expectation getting in the way